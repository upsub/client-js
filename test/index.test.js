/* eslint-env jest */
import ClientES from 'Client'
const ClientCJS = require('Client')

test('Should export the Client', () => {
  expect(ClientES).toMatchSnapshot()
  expect(ClientCJS).toMatchSnapshot()
})
