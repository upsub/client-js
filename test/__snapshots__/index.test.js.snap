// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should export the Client 1`] = `[Function]`;

exports[`Should export the Client 2`] = `
Object {
  "default": [Function],
}
`;
